code/modules/mob/living/emote.dm

```diff
@@ -368,6 +368,13 @@
 	message_mime = "acts out an exaggerated silent sigh."
 	emote_type = EMOTE_VISIBLE | EMOTE_AUDIBLE
 
+/datum/emote/living/sigh/run_emote(mob/living/user, params, type_override, intentional)
+	. = ..()
+	if(!ishuman(user))
+		return
+	var/image/emote_animation = image('icons/mob/species/human/emote_visuals.dmi', user, "sigh")
+	flick_overlay_global(emote_animation, GLOB.clients, 2.0 SECONDS)
```

Comment:
this one also would be nice if you could move the ishuman check to `can_run_emote()`, but it's optional since you aren't implementing that one by default anyways.

Reply:
Wouldn't that mean that other mobs aren't able to use the *sigh emote anymore? The visual is just for humans, since it wouldn't line up correctly with most of the other mobs

Reply:
oh my bad i didn't see that it wasn't a carbon/human emote never mind